druid-shell/src/gtk/mod.rs

```diff
@@ -278,12 +285,11 @@ impl WindowBuilder {
                 }
 
                 Inhibit(true)
-            });
+            }));
         }
 
         {
-            let handle = handle.clone();
-            drawing_area.connect_scroll_event(move |_widget, scroll| {
+            drawing_area.connect_scroll_event(clone!(handle => move |_widget, scroll| {
```

Comment:
can we address this comment from the earlier review, and update one of the comments below?

https://github.com/xi-editor/druid/pull/120/files#r317243827